src/app/components/LiveMediaStream/index.tsx

```diff
@@ -0,0 +1,109 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { memo, useContext, useState } from 'react';
+import Text from '#app/components/Text';
+import { MediaCollection } from '#app/components/MediaLoader/types';
+import MediaLoader, { BumpLoader } from '#app/components/MediaLoader';
+import filterForBlockType from '#app/lib/utilities/blockHandlers';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import mediaIcons from '#psammead/psammead-assets/src/svgs/mediaIcons';
+import styles from './index.styles';
+
+type Props = { mediaCollection: MediaCollection[] | null };
+const DEFAULT_WATCH__NOW = 'Watch Now';
+
+const MemoizedMediaPlayer = memo(MediaLoader);
+
+const LiveMediaStream = ({ mediaCollection }: Props) => {
+  const { translations } = useContext(ServiceContext);
+  const [showMedia, setShowMedia] = useState(false);
+
+  if (mediaCollection == null || mediaCollection.length === 0) {
+    return null;
+  }
+
+  const {
+    media: { watchNow = DEFAULT_WATCH__NOW },
+  } = translations;
+
+  const mediaItem = filterForBlockType(mediaCollection, 'liveMedia');
+
+  const {
+    model: {
+      title,
+      masterbrand: { networkName },
+      synopses: { short },
+      version: { vpid },
+    },
+  } = mediaItem;
+
+  const handleClick = () => {
+    const mediaPlayer = window.mediaPlayers?.[vpid];
+    if (mediaPlayer?.player.paused()) {
+      mediaPlayer?.play();
+      setShowMedia(true);
+    } else {
+      mediaPlayer?.pause();
+      setShowMedia(false);
+    }
+  };
+
+  return (
+    <div css={styles.ComponentContainer}>
+      <BumpLoader />
```

Comment:
I think we can remove `BumpLoader` here as the media player is being loaded on the page anyway.

Means we can remove the `export` from `BumpLoader` as well.